{"uuid": "2cbb8406-cff0-44f3-ac7f-c1e24410df6e", "children": ["2798f911-5237-43b3-8062-8a8f69875482"], "befores": [{"name": "pre_conditions_tc_008", "status": "broken", "statusDetails": {"message": "serial.serialutil.SerialException: could not open port 'COM7': FileNotFoundError(2, 'The system cannot find the file specified.', None, 2)\n", "trace": "  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Scripts\\Demo_Code\\test_001_demo_1.py\", line 157, in pre_conditions_tc_008\n    rly = Relay16CH('COM7')\n  File \"C:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Hardware\\relay_usb.py\", line 6, in __init__\n    self._ser = serial.Serial(port=com_port, baudrate=baud_rate)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialwin32.py\", line 33, in __init__\n    super(Serial, self).__init__(*args, **kwargs)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialutil.py\", line 244, in __init__\n    self.open()\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialwin32.py\", line 64, in open\n    raise SerialException(\"could not open port {!r}: {!r}\".format(self.portstr, ctypes.WinError()))\n"}, "start": 1637780985137, "stop": 1637780986156}], "start": 1637780985137, "stop": 1637780986227}